fatemeh/JS/Basic_Syntax.js

```diff
@@ -0,0 +1,119 @@
+// add number a to b
+function add(a, b) {
+  return a + b;
+}
+
+// flip a coin and log the result (head or tail)
+function flipACoin() {
+  let num = Math.floor(Math.random() * 2);
+
+  switch (num) {
+    case 0:
+      return "head";
+    case 1:
+      return "tail";
+  }
+}
+
+// concat two strings
+function concat(stringA, stringB) {
+  return stringA.concat(stringB);
+}
+
+// 2^x = ?
+function power2(x) {
+  return x * x;
+}
+
+// user object with first name and last name, both capitalized
+// e.g. getUser('hello', 'world') is { firstName: 'Hello', lastName: 'World' }
+function getUser(firstName, lastName) {
+  const newFirstName = firstName
+    .slice(0, 1)
+    .toUpperCase()
+    .concat(firstName.slice(1).toLowerCase());
+  const newLastName = lastName
+    .slice(0, 1)
+    .toUpperCase()
+    .concat(lastName.slice(1).toLowerCase());
+
+  return { newFirstName, newLastName };
```

Comment:
your result for `getUser('hello', 'world');` 
should be `{firstName: 'Hello', lastName: 'World'}`
instead of `{newFirstName: 'Hello', newLastName: 'World'}`.